source/iface/platform.rst

```diff
@@ -152,16 +147,16 @@ the type of information.
      - OpenCL profile
    * - version
```

Comment:
SYCL WG wants us to use namespaces all the way
It's better to change descriptors to `info::platform::<descriptor>` and return types to `std::<return type>`

Reply:
Changed